sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_patch.py

```diff
@@ -113,7 +189,10 @@ def publish_cloud_events(
         :param topic_name: Topic Name. Required.
         :type topic_name: str
         :param body: Cloud Event or array of Cloud Events being published. Required.
-        :type body: ~azure.core.messaging.CloudEvent or list[~azure.core.messaging.CloudEvent]
+        :type body: ~azure.core.messaging.CloudEvent or list[~azure.core.messaging.CloudEvent] or dict[str, Any] or list[dict[str, Any]]
```

Comment:
non-blocking comment: ```suggestion
        :type body: ~azure.core.messaging.CloudEvent or list[~azure.core.messaging.CloudEvent] or dict[str, any] or list[dict[str, any]]
```